### DISCUSSION TITLE
Clear hidden?

### DISCUSSION THREAD

**Discussion by insurw** [cvzi/Metacritic-userscript#25] [Q&A] [answered]
Can't you toggle/clear hidden URL? I can't find anything in settings. Only way is to remove and reinstall the script. Shouldn't hidden URLs be in some list you can edit?

**Comment by cvzi** [comment, marked as answer]
At the moment it is not possible. Maybe I can add an entry to the Tampermonkey menu to clear the current URL. I'll check later if it is easy to add.

**Reply by insurw** [reply]
The only reason I really wanna hide it sometimes is because it's so big so it's in the way. Though I have 125% zoom on practically all sites so it's a bigger issue for me.

Maybe add a minimize?
Maybe make it 1/2 of the current size and scroll for the rest? I.e. only have the Metascore section visible.
Or maybe just make it smaller with same data visible. Though I guess you've tried more compact design and it didn't work/look good.

It's weird though as sometimes (with the same zoom %) there's a scrollbar for the poster section, while sometimes there isn't and the script is ~double the size. 
To replicate: 
- load page
- scroll down to the poster
- reload
Now there's no scrollbar.
- reload
Now there's a scrollbar again.

Sorry, I probably should've just posted an issue instead.

**Reply by cvzi** [reply]
There should be a ❎-symbol that should hide it (without putting it on the blacklist)

**Reply by cvzi** [reply]
I added an option to remove from blacklist and option to shrink/enlarge. Both are in the Tampermonkey menu. 

I can reproduce the problem with the poster. Not sure why it happens. If it is important to you, please open an other issue for it, I might look it later when I have some time.
